sqlglot/enums.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
```

Comment:
@tobymao Where would you like me to put this? I couldn't decide so I figured I would just do this and update with your preference.